db/migrate/20151012010940_create_users.rb

```diff
@@ -3,7 +3,7 @@ def change
     create_table :users do |t|
       t.string :name
       t.string :email
-      t.string :password_digest
+      t.string :password
```

Comment:
In general editing existing migrations is not a good idea, typically its best to create a new migration